flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/HiveTableSource.java

```diff
@@ -40,18 +45,28 @@
 import org.apache.hadoop.hive.metastore.api.StorageDescriptor;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
 import java.sql.Date;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
+import static org.apache.flink.connectors.hive.HiveOptions.TABLE_EXEC_HIVE_INFER_SOURCE_PARALLELISM;
```

Comment:
If possible, I would like to avoid `import static`, it makes reader unclear when reading codes, questions like "why this function will be here in this class" come to mind from time to time.

Reply:
It is a tradeoff, If use it too many times and have a clear name, I prefer to import.

Reply:
But for this case, I think I can remove import.